frontend/app/routes/public/renew/$id/adult-child/children/$childId/information.tsx

```diff
@@ -46,6 +46,11 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
 
 export async function loader({ context: { appContainer, serviceProvider, session }, params, request }: LoaderFunctionArgs) {
   const state = loadRenewAdultSingleChildState({ params, request, session });
+
+  if (!state.isNew) {
+    return redirect(getPathById('public/renew/$id/adult-child/children/$childId/dental-insurance', params));
+  }
+
```

Comment:
FYI TeamCity build fails as a result of this new code:

![image](https://github.com/user-attachments/assets/9b4df0da-503b-4626-a5e1-03042294f9e0)